.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: false
 cache: bundler
 language: ruby
 rvm:
-  - 1.8.7
```

Comment:
Wow, fast response time, @tenderlove!

Maybe we can create a development branch for commits approved for inclusion in the future 2.0 release, and I will send you PRs for merging into that branch?